content/architecture/build/_index.md

```diff
@@ -9,28 +9,32 @@ tags: ["ci", "build", "devops", "pipeline", "pipelines"]
 
 ### Quality Build
 
-Building the docker image, running tests and checking code.  
+When creating a Pull Request, or commiting to a branch in Pull Request, a pipeline is building the applications, running tests and style check/linting.
```

Comment:
skip this part: ", or commiting to a branch in Pull Request," not necessary and adds confusion